content/en/docs/installation/scenarios.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Scenarios"
+linkTitle: "Scenarios"
+weight: 10
+type: "docs"
+---
```

Comment:
suggestion: as noted before, I think this doesn't need to be a separate page; it could just go into the `_index` for `Installation`, and the index page there could be the one-stop-shop for describing different types of installation